repository/SmalltalkCI-Coverage-Core.package/SCICodeCoverage.class/class/coverallsFilesDirectory.st

```diff
@@ -0,0 +1,3 @@
+coveralls
+coverallsFilesDirectory
+	^ (SmalltalkCI getEnv: 'SMALLTALK_CI_BUILD') asFileReference 
```

Comment:
`asFileReference` only works in Pharo, maybe you can reimplement this method in `SCIPharoCodeCoverage` class ?

Reply:
Sure, I've already made the changes you suggested.